lib/features/web_view_etc/map_view.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:google_maps_flutter/google_maps_flutter.dart';
+
+class MapViewScreen extends StatefulWidget {
+  @override
+  _MapViewScreenState createState() => _MapViewScreenState();
+}
+
+class _MapViewScreenState extends State<MapViewScreen> {
+  Completer<GoogleMapController> _controller = Completer();
+
+  static const LatLng _center = const LatLng(45.521563, -122.677433);
```

Comment:
Don't need to write class name and const twice. "static const _center = LatLng(45.521563, -122.677433);" is enough